repos/fdbt-site/src/pages/api/managePurchaseMethod.ts

```diff
@@ -90,8 +92,12 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
 
         if (errors.length > 0) {
             updateSessionAttribute(req, GS_PURCHASE_METHOD_ATTRIBUTE, { inputs: salesOfferPackage, errors });
-
-            redirectTo(res, `/managePurchaseMethod${id ? `?id=${id}` : ''}`);
+            redirectTo(
+                res,
+                `/managePurchaseMethod${
+                    id ? (isCapped ? `isCapped=true&id=${id}` : `?id=${id}`) : isCapped ? '?isCapped=true' : ''
```

Comment:
If we had a unit test for this, the issue would have been caught much earlier. Please can we create one?